src/test/mocks/apis/employees.ts

```diff
@@ -31,20 +31,24 @@ export function handleGetCompanyEmployees(
     GetV1CompaniesCompanyIdEmployeesRequest,
     Employee$Outbound[]
   >,
+  companyId = 'some-company-uuid',
 ) {
-  return http.get(`${API_BASE_URL}/v1/companies/some-company-uuid/employees`, resolver)
+  return http.get(`${API_BASE_URL}/v1/companies/${companyId}/employees`, resolver)
 }
 
-export const getCompanyEmployees = handleGetCompanyEmployees(() =>
-  HttpResponse.json([
-    {
-      uuid: 'some-unique-id',
-      first_name: 'Maximus',
-      last_name: 'Steel',
-      payment_method: 'Direct Deposit',
-    },
-  ]),
-)
+export const getCompanyEmployees = (companyId?: string) =>
```

Comment:
Made this generalizable so a companyId could be provided. This way the knowledge of the "correct" company ID could move out of the mocks and into the tests for simpler reading of the tests.